sgn.conf

```diff
@@ -26,6 +26,8 @@ composable_gen_root_cvterm "generation|TIME:0000072"
 composable_evt_root_cvterm "event|TIME:0000477"
 allow_observation_variable_submission_interface 0
 trait_ontology_db_name SP
+trait_ontology_cv_name solanaceae_trait
+trait_ontology_cvterm_name Solanaceae trait ontology
```

Comment:
configure which cvterm_name to be the main ontology used by the BrAPI endpoints

Reply:
Make sure that these aren't influencing the GET endpoints (user can still specify the ontology dbid they want to fetch from)